src/InputForm.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react'
+import { Button, Cell, Grid, HFlow, TextField, VFlow } from 'bold-ui'
+import React, { useState } from 'react'
+
+function InputForm() {
+  const [formState, setFormState] = useState({
+    input: '',
+  })
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target
+    if (/^\d*$/.test(value)) {
```

Comment:
```suggestion
    if (isNumber(value)) {
```
Seria legal colocar numa função, regex não é dos mais legíveis 😅